src/main/java/com/exadel/discount/service/impl/JwtServiceImpl.java

```diff
@@ -76,6 +93,18 @@ public String generateRefreshToken(UserDetails userDetails) {
      * @param role           it will be set in a role claim.
      * @return a built JWT.
      */
+    private String buildToken(String subject, long expirationTime, String role, String id) {
```

Comment:
instead of role and id add map of claims as input parameter and on building token just add claim for each map element

Reply:
Done